### ISSUE TITLE
System.ArgumentOutOfRangeException: Schema mismatch for input column 'Label': expected KeyType, got Text

### ISSUE THREAD

**Issue by suhaschitade** [dotnet/machinelearning-samples#347]
Problem encountered on https://dotnet.microsoft.com/learn/machinelearning-ai/ml-dotnet-get-started-tutorial/code
Operating System: macos

Provide details about the problem you are experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

Unhandled Exception: System.ArgumentOutOfRangeException: Schema mismatch for input column 'Label': expected KeyType, got Text
Parameter name: inputSchema
   at Microsoft.ML.Transforms.KeyToValueMappingEstimator.GetOutputSchema(SchemaShape inputSchema)
   at Microsoft.ML.Data.EstimatorChain`1.GetOutputSchema(SchemaShape inputSchema)
   at Microsoft.ML.Data.EstimatorChain`1.Fit(IDataView input)
   at myFirstML.Program.Main(String[] args) in /Users/suhaschitade/dotnetspace/ml/myFirstML/Program.cs:line 42

**Comment by CESARDELATORRE** [comment]
What version of ML.NET are you using?
The tutorial is still using ML.NET 0.11.

I think that’s a breaking change in 1.0.0-preview 
We’ll update the tutorial to 1.0.0-preview pretty soon.

**Comment by CESARDELATORRE** [comment]
Closing this issue. The Getting Started tutorial has already been migrated to ML.NET 1.0.0-preview.
Please confirm it is working for you. Other than that, let's re-open this issue, ok?

Thanks! 👍

**Comment by suhaschitade** [comment]
Please open this issue, same behaviour observed after upgrading ML version, is this really production ready?

**Comment by CESARDELATORRE** [comment]
@suhaschitade - You must be doing any step wrong. I just went through the whole 'Getting Started' tutorial from a Mac and it works with no issues. 

Note that the source-code is slightly different in ML.NET 1.0.0-preview compared to ML.NET 0.11.

Check it out in this screenshot:

![image](https://user-images.githubusercontent.com/1712635/55677352-b7995d00-589a-11e9-9bd6-a0a29ed354e4.png)

Closing the issue. Feel free to open it if you still are having issues, ok?

Thanks,
